app/templates/components/hero.html

```diff
@@ -1,14 +1,19 @@
-<div class="hero-section">
-  <h1 class="pt-4 pb-2 font-bold text-3xl">{{ translations.get("title") }}</h1>
-  <h2 class="font-bold leading-tight text-2xl mt-0 mb-2 text-pink-500">
-    {{ translations.get("disclaimer") }}
-  </h2>
-  <p class="py-4">
-    Lorem ipsum dolor sit, amet consectetur adipisicing elit. Assumenda soluta
-    sed consequatur rem saepe dolores voluptates recusandae explicabo officiis
-    ipsa nesciunt, aliquid placeat dolor consectetur ullam tenetur impedit eaque
-    quam quas esse facilis voluptate ducimus voluptatem. Itaque dolores
-    similique, eaque animi magni, quaerat consectetur sit suscipit quas nam
-    molestias culpa.
-  </p>
+<div
+    class="hero-section fixed top-0 left-0 bg-corporate-blue text-white w-screen h-3/5 -z-10 py-10 px-mobile tablet:px-desktop desktop:max-w-hero desktop:h-full"
+>
+    <div
+        class="w-full h-full max-w-4xl mx-auto flex flex-col justify-between desktop:justify-start"
+    >
+        <img
+            class="w-44 self-end desktop:mb-16"
+            src="{{ url_for('static', path='images/TSB-Logo-outline-weiss.svg') }}"
+            alt="{{ translations.hero.imageAlt }}"
+        />
+        <div class="w-80 desktop:self-center">
+            <h1 class="pt-4 pb-2 font-bold pb-0">
```

Comment:
`pb-2` is canceling out `pb-0`, so you probably want to remove `pb-0` here!

Reply:
si claro!